### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by AndriiTorhashov**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,12 +6,253 @@
...
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+      class="form"
+      >
```
Comment (Viktor-Kost) [inline src/index.html:26]:
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="post"
      class="form"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -6,12 +6,253 @@
...
+          <label>
+            Surname:
+            <input 
+            type="text" 
+            name="sname" 
+            required 
+            autocomplete="off"
+            >
+          </label> 
+        </div>
```
Comment (Viktor-Kost) [inline src/index.html:42]: Please check code styling moments ALL OVER the page. Especially point 2-3
[Recommendations](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
```suggestion
      <fieldset class="group">
        <legend>Personal information</legend>
        <div class="simple">
          <label>
            Surname:
            <input 
              type="text" 
              name="sname" 
              required 
              autocomplete="off"
            >
          </label> 
        </div>
```

File: src/index.html [hunk 3]
```diff
@@ -6,12 +6,253 @@
...
+        <div class="line"></div>
```
Comment (Viktor-Kost) [inline src/index.html:43]: No needs to add an html element to make a styling issue. 
Use css

File: src/index.html [hunk 4]
```diff
@@ -6,12 +6,253 @@
...
+          <label>
+            Name:
+            <input 
+            type="text" 
+            name="name" 
+            required 
+            autocomplete="off"
+            >
+          </label>
+        </div>
```
Comment (Viktor-Kost) [inline src/index.html:54]: it is a redundant `div` wrapper. You can add styles to `label`. Remove `div` at other similar places as well

File: src/index.html [hunk 5]
```diff
@@ -6,12 +6,253 @@
...
+            value="Yes"
+            >
+            <label for="yes">Yes</label>
+            <input type="radio" 
+            id="no" 
+            name="cat" 
+            value="No"
+            >
+            <label for="no">No</label>
+          </label>
```
Comment (Viktor-Kost) [inline src/index.html:143]: Here you need two labels. 
labels should wrap ONLY inputs. 
Do you love cats? should not be wrapped into a label

File: src/index.html [hunk 6]
```diff
@@ -6,12 +6,253 @@
...
+            name="accept" 
+            required
+            >
+          </label>
+        </div>
+      </fieldset>
+    
+
+    
+      <fieldset class="group">
```
Comment (Viktor-Kost) [inline src/index.html:95]: is too big distance between siblings
```suggestion
      </fieldset>

      <fieldset class="group">
```

Comment (Viktor-Kost) [inline src/index.html:95]: is too big distance between siblings. Check all over the page
```suggestion
      </fieldset>

      <fieldset class="group">
```

File: src/index.html [hunk 7]
```diff
@@ -6,12 +6,253 @@
...
 </html>
+
+
+
+
+  
+ 
+
+  
+
```
Comment (Viktor-Kost) [inline src/index.html:258]: delete these empty lines

File: src/style.css [hunk 8]
```diff
@@ -1 +1,21 @@
...
+  margin-bottom: 10px;
+  display: block;
+  
+}
+
+.group {
+  
+  margin-bottom: 20px;
+  
+}/* styles go here */
```
Comment (Viktor-Kost) [inline src/style.css:12]: delete unused comments

File: src/style.css [hunk 9]
```diff
@@ -1 +1,21 @@
 /* styles go here */
```
Comment (Viktor-Kost) [inline src/style.css:1]: delete unused comments

File: src/style.css [hunk 10]
```diff
@@ -1 +1,21 @@
...
+.simple {
+  margin-bottom: 10px;
+  display: block;
+  
+}
+
+.group {
+
```
Comment (Viktor-Kost) [inline src/style.css:9]: extra empty line

File: src/style.css [hunk 11]
```diff
@@ -1 +1,21 @@
...
+.group {
+  
+  margin-bottom: 20px;
+  
+}/* styles go here */
+
+.line {
+  height: 1 px;
+  display: block;
+}
```
Comment (Viktor-Kost) [inline src/style.css:17]: ``
```suggestion
```

Comment (Viktor-Kost) [inline src/style.css:17]: ``
```suggestion
```

File: src/index.html [hunk 12]
```diff
@@ -6,12 +6,253 @@
...
+          </label>
+        </div>
+      </fieldset>
+    
+   
+      <div class="simple">
+        <label>
+          <input 
+          type="submit" 
+          id="sb"
```
Comment (Viktor-Kost) [inline src/index.html:241]: why do you need this id?

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Viktor-Kost) [reply src/index.html:26]: is not fixed

File: src/index.html (hunk 2 above)
Comment (Viktor-Kost) [reply src/index.html:42]: is not fixed

File: src/index.html (hunk 3 above)
Comment (Viktor-Kost) [reply src/index.html:43]: is not fixed

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Pay attention to unresolved conversations. 
I guess you haven't committed changes

If you need any help - ask at fe_chat

**Comment by AndriiTorhashov** [comment]
Я перезалив вже завдання, де всі помилки виправив. Це завдання тести
пройшло. Можливо я якось не правильно його відправив на гіт хаб, але наче
все завантажилось.
пт, 30 груд. 2022 р. о 17:10 Viktor-Kost ***@***.***> пише:

**Pull request closed without merging**